### PR TITLE
[REVIEW] tests to bug 1788

### PR DESCRIPTION
**Pull request by iariasleon**
#### Regression test
```
                    SUMMARY:
 --------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|                                                       name                                                           | passed | failed | skipped |  total  | duration(s) |
 --------------------------------------------------------------------------------------------------------------------------------------------------------------------------
| components\ngsiv2\entities\update_existing_entity_attributes\update_existing_entity_attributes.feature               |    111 |      0 |       1 |     112 |      34.470 |
| components\ngsiv2\entities\update_existing_entity_attributes\update_existing_entity_attributes_part2.feature         |    305 |      0 |       0 |     305 |     207.360 |
| components\ngsiv2\entities\update_existing_entity_attributes\update_existing_entity_attributes_part3.feature         |    206 |      0 |      16 |     222 |      79.015 |
| components\ngsiv2\entities\update_existing_entity_attributes\update_existing_entity_attributes_part4.feature         |      2 |      0 |       0 |       2 |       0.776 |
| components\ngsiv2\entities\update_existing_entity_attributes\update_existing_entity_attributes_traces_in_log.feature |      1 |      0 |       0 |       1 |       3.154 |
 --------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|  Date: Thursday, November 10, 2016 16:28:24
 -------------------------------------------------
5 features passed, 0 failed, 0 skipped
625 scenarios passed, 0 failed, 17 skipped
4488 steps passed, 0 failed, 96 skipped, 0 undefined
Took 5m24.775s
```
@crbrox

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature [hunk 1]
```diff
@@ -445,15 +445,14 @@ Feature: update an attribute by entity ID if it exists using NGSI v2. "PATCH" -
...
-    #  | attributes_value | 45          |
       | metadatas_name  | my_meta     |
       | metadatas_value | 5678        |
     When update attributes by ID "room" if it exists and with "normalized" mode
     Then verify that receive an "No Content" http code
     And verify that an entity is updated in mongo
 
-  @attribute_metadata_replace @BUG_1788 @skip
-  Scenario:  update an attribute by entity ID using NGSI v2 and append a new metadata without metadata type
+  @attribute_metadata_replace_2 @BUG_1788
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature]: `@attribute_metadata_replace_2` is not a very good name...

Maybe this should be renamed to `@attribute_metadata_replace_with_md_type` and `@attribute_metadata_replace` to `@attribute_metadata_replace_without_md_type` ?

File: test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature [hunk 2]
```diff
@@ -479,7 +478,6 @@ Feature: update an attribute by entity ID if it exists using NGSI v2. "PATCH" -
     And properties to entities
       | parameter        | value       |
       | attributes_name  | temperature |
-      | attributes_value | 45          |
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature]: This means that the attribute has no value and not ype? I mean, something like this at API level:

PUT /v2/entities/room/attrs/temperature

{
  "metadata": {
    "my_meta": { "value": 5687, "type": "nothing" }
  } 
}

File: test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature]: fixed in dd1d595

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature]: fixed in dd1d595

**Comment by fgalan** [comment]
LGTM

**Pull request merged**